sdks/go/examples/wordcount/wordcount.go

```diff
@@ -123,6 +125,11 @@ func extractFn(ctx context.Context, line string, emit func(string)) {
 		empty.Inc(ctx, 1)
 	}
 	for _, word := range wordRE.FindAllString(line, -1) {
+		// increment the counter for small words if length of words is
+		// less than small_word_length
+		if len(word) < *small_word_length {
```

Comment:
Flags aren't available on workers. They'd only have their default values. `extractFn` needs to change to a structural DoFn (like it is in the other example being updated) so the flag functions properly.